azure-pipelines.yml

```diff
@@ -130,15 +130,15 @@ jobs:
       skipTests: true
       targetArchitecture: armel
 
-# - template: /eng/jobs/linux-build.yml
-#   parameters:
-#     additionalMSBuildArgs: /p:OutputRid=rhel.6-x64
-#     displayName: Build_Linux_x64_FreeBSD
-#     dockerImage: microsoft/dotnet-buildtools-prereqs:centos-6-376e1a3-20174311014331
-#     osGroup: FreeBSD
-#     portableBuild: true
-#     skipTests: true
-#     targetArchitecture: x64
+- ${{ if ne(variables['System.TeamProject'], 'public') }}:
+  - template: /eng/jobs/linux-build.yml
+    parameters:
+      displayName: Build_Linux_x64_FreeBSD
```

Comment:
Since FreeBSD is its own OS family (not related to Linux), should this just be `Build_FreeBSD_x64`?

Reply:
@rakeshsinghranchi Or, actually, I do think this change is worthwhile. (Also brought up in https://github.com/dotnet/core-setup/pull/4936#discussion_r246215809.)

Reply:
I'll submit a new PR with these changes.